packages/client/tests/functional/multi-schema/_matrix.ts

```diff
@@ -0,0 +1,18 @@
+import { defineMatrix } from '../_utils/defineMatrix'
+
+export default defineMatrix(() => [
+  [
+    {
+      provider: 'postgresql',
+    },
+
+    {
+      provider: 'sqlserver',
+    },
+  ],
+  [
+    {
+      mapTable: 'true',
```

Comment:
Doesn't this mean that `mapTable` is always true? Perhaps you could also make it a boolean instead of a string.

Reply:
Using the matrix here so we run the suite first without this item and then second with it, this is a typing issue will update the types to allow booleans too.